Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY Pipfile Pipfile
 COPY Pipfile.lock Pipfile.lock
```

Comment:
Was this not needed with `pyproject.toml`?

Reply:
Missed that one, yeah not needed.